ibr/binToJs.js

```diff
@@ -1,21 +1,113 @@
 /**
- * @fileoverview This file takes a binary wire format ibr file, decode the information and store data in sessionStorage for visualization use
+ * @fileoverview This file takes a binary wire format ibr file, decode the
+ information and store data in sessionStorage for visualization use.
  * @author shuanglihtk@google.com (Shuang Li)
  */
 
-import {OrbitControls} from './node_modules/three/examples/jsm/controls/OrbitControls.js';
+import {OrbitControls} from
+  './node_modules/three/examples/jsm/controls/OrbitControls.js';
 
+/**
+ * Create a new HTML Label Tag based on given name string and attach it to
+ * given parent tag.
+ * @param {string} tagName Type of tag to be created.
+ * @param {string} name InnerHTML of the Label tag.
+ * @param {tag} parentTag Tag to be attached to by the newly created label
+  tag.
+ * @param {string} [forId=undefined] ID to be set as the value of the newly
+  created label tag's for attribute. Needed if Label tag is created for a
+  checkbox.
+ * @return {tag} Newly created Label tag.
+ */
+function createLabel(tagName, name, parentTag, forId=undefined) {
+  const label = document.createElement(tagName);
```

Comment:
there's several instances of document.createElement. couldn't we just include that in the html and use jQuery to reference them instead?

this would make it easier to look at the html file debug since we're not creating so many elements on the fly

Reply:
Will open a new PR to refactor all functions.